File: Serene3/Serene3.Web/Modules/com/orderdetials/orderdetialsForm.cs
using Serenity.ComponentModel;
using System;

namespace Serene3.com.Forms;

[FormScript("com.orderdetials")]
[BasedOnRow(typeof(orderdetialsRow), CheckNames = true)]
public class orderdetialsForm
{
    public int Productid { get; set; }
    public decimal Totalprice { get; set; }
    public int Quantity { get; set; }
    public int Orderid { get; set; }
    public decimal Price { get; set; }
    public DateOnly Entitydata { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/orderdetialsForm.ts
import { DecimalEditor, initFormType, IntegerEditor, PrefixedContext, ServiceLookupEditor, StringEditor } from "@serenity-is/corelib";

export interface orderdetialsForm {
    Productid: ServiceLookupEditor;
    Totalprice: DecimalEditor;
    Quantity: IntegerEditor;
    Orderid: ServiceLookupEditor;
    Price: DecimalEditor;
    Entitydata: StringEditor;
}

export class orderdetialsForm extends PrefixedContext {
    static readonly formKey = 'com.orderdetials';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!orderdetialsForm.init) {
            orderdetialsForm.init = true;

            var w0 = ServiceLookupEditor;
            var w1 = DecimalEditor;
            var w2 = IntegerEditor;
            var w3 = StringEditor;

            initFormType(orderdetialsForm, [
                'Productid', w0,
                'Totalprice', w1,
                'Quantity', w2,
                'Orderid', w0,
                'Price', w1,
                'Entitydata', w3
            ]);
        }
    }
}